README.md

```diff
@@ -90,11 +113,32 @@ def mystery5(s)
 end
 ```
 
-- What is mystery5("hi")?
-- What is mystery5("")?
-- What is mystery5("Hi, there!")?
+- What is mystery5("hi")? "**"
+- What is mystery5 ""
+- What is mystery5("Hi, there!")? "**********"
 - _Added Fun: How could we make only alphabetic characters to be changed to stars?_
 
+We could make change it to the following,
+
+```
+chars = ('a'..'z').to_a + ('A'..'Z').to_a
+
+def mystery5(s)
+
+chars = ('a'..'z').to_a + ('A'..'Z').to_a
+
+  if s.length == 0
+    return ""
+  else
+    if chars.include? s[0]
+      return "*" + mystery5(s[1..-1])
+    else
+      return mystery5(s[1..-1])
+    end
+  end
+end
+```
```

Comment:
Nice job! A couplet things to consider here. 

1) Regex would make this potentially a more sophisticated solution, but yours is maybe easier to read.

2) Is your assumption that we would want to strip out all non-alpha characters? That's what your solution would produce ==> mystery5("Hi, there!") would be "*******", but what if we wanted it to be "**, *****!" which is what I intended. I will make these instructions more clear for my next iteration of this assignment.